acceptance/bundle/deploy/experimental-python/my_project/databricks.yml

```diff
@@ -0,0 +1,9 @@
+bundle:
```

Comment:
why have my_project subdirectory, is it essential for this test? we don't use this pattern in other tests.

Reply:
Agree, flat directories are more simple